lib/Graph/Nodes.cpp

```diff
@@ -1531,6 +1531,7 @@ bool LengthsSumNode::verify() const {
   }
 
 DEFINE_BATCHED_REDUCTION_VERIFICATION(BatchedReduceAdd)
+DEFINE_BATCHED_REDUCTION_VERIFICATION(BatchedReduceAnd)
```

Comment:
In this case we only ever expect to support Bool input/outputs right? Maybe we can also verify that is the case?

Reply:
I've removed the macro and made it's own verification function with the boolean check